Dockerfile

```diff
@@ -80,6 +80,13 @@ RUN echo "source activate gatk" > /root/run_unit_tests.sh && \
     echo "ln -s /gatkCloneMountPoint/build/ /gatkCloneMountPoint/scripts/docker/build" >> /root/run_unit_tests.sh && \
     echo "cd /gatk/ && /gatkCloneMountPoint/gradlew -Dfile.encoding=UTF-8 -b /gatkCloneMountPoint/dockertest.gradle testOnPackagedReleaseJar jacocoTestReportOnPackagedReleaseJar -a -p /gatkCloneMountPoint" >> /root/run_unit_tests.sh
 
+# TODO: Determine whether we actually need this.  For now it seems to be required because the version of libstdc++ on
+# TODO: the gatk base docker is out of date (maybe?)
```

Comment:
Our `ah_var_store` branch is out of date with `master` again; is it possible that this issue would go away once we refresh?

Reply:
It might, yeah.  I know the base image in master was recently updated to Ubuntu 22.04, which might have the more recent version of the necessary library.

Reply:
I know having this step in the docker resolves the issue, so I'd say it's very low priority.  We could probably just remove it and run a quick test when the refresh happens.